packages/tools/src/LernaTool.ts

```diff
@@ -59,10 +55,10 @@ export const LernaTool: Tool = {
     const rootDir = path.resolve(directory);
 
     try {
-      const lernaJson = await fs.readJson(path.join(rootDir, "lerna.json"));
-      const pkgJson = (await fs.readJson(
+      const lernaJson = await readJson(rootDir, "lerna.json") as LernaJson;
+      const pkgJson = JSON.parse((await fs.promises.readFile(
```

Comment:
this could use the util that you introduced, right?

Reply:
yep. done